playwright/e2e/share-dialog/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+import { test as base } from "../../element-web-test";
+
+/**
+ * Set up for share dialog tests.
+ */
+export const test = base.extend<{
+    roomName?: string;
```

Comment:
Hmm, how do you get it?

I can remove the room name but I still need it to create the room:
```ts
room: async ({ app, user, bot }, use) => {
    const roomId = await app.client.createRoom({ name: "Alice room", invite: [bot.credentials.userId] });
    await use({ roomId });
},
```

Reply:
Sure, you need it for room creation, but it could just be a const in the file, not sure why its a fixture as nothing else uses it and you can always get it as `room.name`